### PR TITLE
feat(ci): add earthfile for voices fronted

### PR DESCRIPTION
**Pull request by minikin**
- Prepare Catalyst Voices Web for deployment. 
- Unblocks https://github.com/input-output-hk/catalyst-world/issues/183
Screenshot 2024-02-23 at 1 27 47 PM

### CODE REVIEW

File: catalyst_voices/Earthfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+VERSION --try --global-cache 0.8
```
Comment (jmgilman) [inline catalyst_voices/Earthfile:1]: Looks like CI is failing, try setting this to `0.7`

**Review by jmgilman — CHANGES_REQUESTED** [review]

File: catalyst_voices/Earthfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+docker:
+    FROM +deps
+    FROM nginx:stable-alpine3.17
+
+    COPY +build/dist /app
+    COPY ./nginx.conf /etc/nginx/nginx.conf
+
+    EXPOSE 80
+
+publish:
```
Comment (stevenj) [inline catalyst_voices/Earthfile:45]:
```suggestion
package:
```
this should be `package:`  to align with CI.  `publish` actually publishes a prebuilt package, whereas this is just building the package ready to publish.

File: catalyst_voices/Earthfile [hunk 3]
```diff
@@ -0,0 +1,50 @@
+VERSION --try --global-cache 0.7
+
+deps:
+    FROM debian:stable-slim
+    RUN apt-get update
+    RUN apt-get install -y curl git unzip bash
+    WORKDIR /frontend
+
+# Download and set-up flutter
+flutter:
```
Comment (minikin) [inline catalyst_voices/Earthfile:10]: FYI @jmgilman we will extract this into catalyst-ci in another task.

**Comment by minikin** [comment]
FYI @jmgilman we will extract `flutter` install and config into catalyst-ci in another task.

**Review by stevenj — CHANGES_REQUESTED** [review]
Looks good, except for the one suggestion.

**Review by stevenj — APPROVED** [review]

**Pull request merged**